emva1288/process/data.py

```diff
@@ -119,14 +119,14 @@ def _get_temporal(self, data):
             # get data for dark image
             d = self._get_temporal_data(data[t][0.0])
             u_ydark.append(d['mean'])
-            s2_ydark.append(d['var'])
+            s2_ydark.append(d['var'] - d['dmean'])
 
             for p in photons[1:]:
                 # For each photon count, get the data
                 u_p.append(p)
                 d = self._get_temporal_data(data[t][p])
                 u_y.append(d['mean'])
-                s2_y.append(d['var'])
+                s2_y.append(d['var'] - d['dmean'])
```

Comment:
Don't you need a factor of 1/2 here and in line 122?  You have it at line 180 but it is present in eqn 18

Reply:
If I see correctly, `d` is assigned from `_get_temporal_data()` which is the calc from line ~180. So no ;-)